spidermon/contrib/scrapy/monitors.py

```diff
@@ -522,6 +530,78 @@ def test_execution_time(self):
         self.assertLess(duration.total_seconds(), max_execution_time, msg=msg)
 
 
+@monitors.name("Jobs Comparison Monitor")
+class JobsComparisonMonitor(BaseStatMonitor):
+    """
+    .. note::
+       This monitor is useful when running jobs in
+       `Zyte's Scrapy Cloud <https://www.zyte.com/scrapy-cloud/>`_.
+
+    Check for a drop in scraped item count compared to previous jobs.
+
+    You need to set the number of previous jobs to compare, using ``SPIDERMON_JOBS_COMPARISON``.
+    The default is ``0`` which disables the monitor. We use the average of the scraped items count.
+
+    You can configure which percentage of the previous item count is the minimum acceptable, by
+    using the setting ``SPIDERMON_JOBS_COMPARISON_THRESHOLD``. We expect a float number between
+    ``0.0`` and ``1.0``. The default is ``0.8``.
```

Comment:
**suggestion** By default, our built-in monitors (see `ItemCountMonitor`, `ErrorCountMonitor`, etc) don't have a default value for the threshold, but they raise a `NotConfigured` if the setting is not provided. I'd prefer to keep it the same with this monitor, so we don't have a default value.